src/Manifest/Manifest.router.ts

```diff
@@ -77,12 +87,34 @@ export class ManifestRouter extends Router {
 
   async getProjectManifest(req: AuthRequest) {
     const id = server.extractFromReq(req, 'id')
+
     const body = await new S3Project(id).readFileBody(MANIFEST_FILENAME)
     if (body) {
       return JSON.parse(body.toString())
     }
   }
 
+  async getManifests(req: AuthRequest) {
+    let eth_address = req.auth.ethAddress
+
+    const projectSearcher = new SearchableProject(req)
+    const projects = await projectSearcher.searchByEthAddress(eth_address)
+
+    if (projects) {
+      let manifests: any[] = []
+      for (const project of projects.items) {
+        const body = await new S3Project(project.id).readFileBody(
+          MANIFEST_FILENAME
+        )
+
+        if (body) manifests.push(JSON.parse(body.toString()))
+      }
+
+      if (manifests) return manifests
+    }
+    return false
```

Comment:
maybe we could return `[]` instead of `false` here? We could move the `const manifest: Manifest[] = []` outside, before the `if (projects) { ... }` block, and just `return manifests` at the end of the function.

Also, this is missing the `?creation_coords` query param, right? Should we add it before returning the `manifests` array, something like:

```
const creationCoords: string | null = server.extractFromReq(req, 'creation_coords')
if (creationCoords) {
  const project = projects.items.find(project => project.creation_coords === creationCoords)
  const manifest = manifests.find(manifest => manifest.project.id === project.id) || null
  return manifest
}
```

Reply:
Ok, I will change it to return an empty array.

For the creation coords part, it is not needed,

with this part of the code
`
    const projectSearcher = new SearchableProject(req)
    const projects = await projectSearcher.searchByEthAddress(eth_address)`
    
    You don't need to specify for creation coords, you just can filter by any parameter that you want from the project

Reply:
ahh okay, i didn't realize that last part, cool :)